doc/advanced_configuration.rst

```diff
@@ -312,3 +312,21 @@ the example script.
 
 
 .. _regular expressions: https://docs.python.org/2/library/re.html
+
+
+Pandoc for converting rst to markdown for notebook downloads
+============================================================
+
+Sphinx uses restructured text while Jupyter notebooks uses markdown for
+markup of the text. Therefore, restructured text need to be translated to
```

Comment:
Due to this mismatch in input formats, the generated Jupyter notebooks may not display correctly if you have heavily used the formatting syntax of restructured text. Sphinx-Gallery provides a minimal parser to transform between this two formats suitable for basic use cases and is the default parse. Sphinx-Gallery is nevertheless aware of pypandoc and if found in your system it will be used to convert between  this formats. To install pypandoc in your system you can use the following commands:


I'm not so good with writing. These paragraphs might need some extra reviews to give a clear and easy to understand message.